### DISCUSSION TITLE
Display # of installed tweaks on repo

### DISCUSSION THREAD

**Discussion by worldlydev** [wstyres/Zebra#956] [Ideas]
https://www.reddit.com/r/jailbreak/comments/fj3wm4/request_small_tick_next_to_repos_where_you_have/

![F0B3441A-F0EC-406A-85DC-9153A0C049FC](https://user-images.githubusercontent.com/51668635/76758219-93701c80-6789-11ea-86f5-037efb7e7603.jpeg)

**Comment by luptakmichal** [comment]
Yes, please.

**Comment by worldlydev** [comment]
The #installed tweak in Cydia:
![46963B2C-CAB3-4226-B1AC-2FB19A65DD63](https://user-images.githubusercontent.com/51668635/79164337-262fc580-7de1-11ea-841c-0ff551b9f475.png)

**Comment by worldlydev** [comment]
Or like the Zenumbra tweak:
![F6F5B0A9-AA19-48E8-BEE2-C893B651DE87](https://user-images.githubusercontent.com/51668635/82142034-d5751780-9839-11ea-9186-f2699e223c48.png)
